metadata-models/src/main/pegasus/com/linkedin/ml/metadata/MLModelDeploymentProperties.pdl

```diff
@@ -35,5 +35,8 @@ record MLModelDeploymentProperties includes CustomProperties, ExternalReference
   /**
    * Status of the deployment
    */
+  @Searchable = {
+    "fieldName": "deploymentStatus"
+  }
```

Comment:
do you want a backfill job or good to just move forward with this?

Reply:
no backfill job, we will proceed forward with this for now. we can instruct the customer on how to do a reindexing of the targeted entities they need